gcs.tf

```diff
@@ -0,0 +1,16 @@
+resource "google_storage_bucket" "source_bucket" {
+  name                        = "source-${var.name}-bucket"
+  force_destroy               = true
+  location                    = upper(var.region)
+  uniform_bucket_level_access = true
+
+  labels = {
+    project = data.google_project.project.name,
+  }
+}
+
+resource "google_storage_bucket_object" "archive" {
+  name   = "${data.archive_file.source_archive.output_md5}.zip"
```

Comment:
It may not be needed, but I ended up doing `${lower(replace(base64encode(data.archive_file.census-splunk-slack-alerts-org.output_md5), "=", ""))}.zip` as I wasn't sure all the chars in output_md5 were safe to go in a filename

Reply:
Doesn't hurt to be safe. I'll get that in.  Thanks, @kierenbeckett.